lib/Syntax/CMakeLists.txt

```diff
@@ -4,19 +4,19 @@ else()
   set(SWIFT_GYB_FLAGS --line-directive "\'#line" "%(line)d" "\"%(file)s\"\'")
 endif()
 
-add_swift_host_library(swiftSyntax STATIC
+add_swift_host_library(legacySwiftSyntax STATIC
```

Comment:
Let's not end up like the legacy pass manager, eh? ;)

Reply:
I don't know if we actually need this, given that CMake has scoping for targets. But it's fine if we need to do it---this should be going away soonish.